python/orca/test/bigdl/orca/learn/ray/pytorch/test_estimator_pyspark_backend.py

```diff
@@ -391,6 +395,29 @@ def get_optimizer(model, config):
         state = estimator.get_state_dict()
         assert state['models'][0]['fc1.weight'].item() == 0.25
 
+    def test_checkpoint_callback(self):
+        from bigdl.orca.learn.pytorch.callbacks.model_checkpoint import ModelCheckpoint
+        sc = OrcaContext.get_spark_context()
+        rdd = sc.range(0, 100)
+        df = rdd.map(lambda x: (np.random.randn(50).astype(np.float).tolist(),
+                                [int(np.random.randint(0, 2, size=()))])
+                     ).toDF(["feature", "label"])
+
+        estimator = get_estimator(workers_per_node=2, model_dir=self.model_dir,
+                                  log_level=logging.DEBUG)
+
+        callbacks = [
+            ModelCheckpoint(filepath=os.path.join(self.model_dir, "test-{epoch}"),
+                            save_weights_only=True)
+        ]
+        estimator.fit(df, batch_size=4, epochs=2,
+                      callbacks=callbacks,
+                      feature_cols=["feature"],
+                      label_cols=["label"])
+        estimator.evaluate(df, batch_size=4,
+                           feature_cols=["feature"],
+                           label_cols=["label"])
+
```

Comment:
maybe checking if the checkpoint file exists?

Reply:
Sure. Thanks for the review.